src/annotations/api/index.test.ts

```diff
@@ -1,52 +1,191 @@
+
+    // Starts with question mark for query string
+    expect(queryString[0]).toEqual('?')
+
+    // Handles spaces
+    expect(queryString.indexOf(' ')).toEqual(-1)
+    expect(queryString.includes('default%20stream')).toBeTruthy()
+
+    // Only uses ? symbol in beginning, following queryString params use &
+    expect(queryString.lastIndexOf('?')).toEqual(0)
+    expect(queryString.split('').filter(a => a === '&').length).toEqual(3)
+
+    // Handles encoding of object params for stickers
+    expect(queryString.includes('&stickerIncludes[foo]=bar')).toBeTruthy()
+  })
+})
 describe('annotations api calls', () => {
   describe('POST - write annotations api calls', () => {
     it('can write an annotation successfully and return the array of annotations with camelcased response properties', async () => {
       const data = [
         {
           summary: 'GO PACK GO',
-          'start-time': Date.now(),
-          'end-time': Date.now(),
+          start: Date.now().toString(),
+          end: Date.now().toString(),
           stream: 'Lambeau Field',
         },
       ]
 
-      const transformedData = [
+      mocked(axios.post).mockImplementationOnce(() =>
+        Promise.resolve({status: 200, data})
+      )
+
+      const response = await writeAnnotation(data)
+
+      expect(axios.post).toHaveBeenCalledTimes(1)
+
+      expect(response).toEqual(data)
+    })
+
+    it('handles an error and returns the error message', async () => {
+      const data = [
         {
           summary: 'GO PACK GO',
-          startTime: Date.now(),
-          endTime: Date.now(),
+          start: Date.now().toString(),
+          end: Date.now().toString(),
           stream: 'Lambeau Field',
         },
       ]
+      const message = 'OOPS YOU DONE MESSED UP SON'
+
       mocked(axios.post).mockImplementationOnce(() =>
-        Promise.resolve({status: 200, data})
+        Promise.reject(new Error(message))
       )
 
-      const response = await writeAnnotation([{summary: 'Hey there'}])
+      await expect(writeAnnotation(data)).rejects.toThrow(message)
+    })
+  })
 
-      expect(axios.post).toHaveBeenCalledTimes(1)
+  describe('GET - retrieve annotations api calls', () => {
+    const annotationResponse = [
+      {
+        stream: 'Lambeau Field',
+        annotations: [
+          {
+            start: Date.now().toString(),
+            end: Date.now().toString(),
+            summary: 'Go Pack Go',
+          },
+        ],
+      },
+      {
+        stream: 'default',
+        annotations: [
+          {
+            start: '2020-01-12T23:13:47Z',
+            end: Date.now().toString(),
+            summary: 'blah blah',
+          },
+        ],
+      },
+    ]
 
-      expect(response).toEqual(transformedData)
+    it('retrieves annotations and returns them categorized by annotation stream', async () => {
+      mocked(axios.get).mockImplementationOnce(() =>
+        Promise.resolve({data: [annotationResponse[0]]})
```

Comment:
I think it would make this test a little bit more self documenting and readable if you pulled this out into a variable like:

```ts
const [lambeauResponse] = annotationResponse
```

then down below it would be like:

```ts
expect(response).toEqual(lambeauResponse)
```